docs/assets/scss/components/_search.scss

```diff
@@ -70,15 +87,19 @@ body > #suggestions{
   outline: 0;
 }
 
-#suggestions div:not(:first-child) {
+#suggestions>div:not(:first-child) {
   border-top: 1px dashed $gray-200;
 }
 
-#suggestions div:first-child {
+#suggestions * {
+  box-sizing: unset;
+}
```

Comment:
Can you comment what is the reason just in case?

It is unlikely this will be a problem since the content in the page is always ours so you can skip the comment and the next comment for the `object` selector below.

Reply:
In fact we don't need this. I tested something with this and now I removed it. Thanks!